setup.py

```diff
@@ -84,17 +83,10 @@
             "sphinx-rtd-theme",
             "sphinxcontrib-apidoc",
         ],
-        "camera-obs": [
-            "Panda3D==1.10.9",
-            "panda3d-gltf==0.13",
-        ],
-        "ros": [
-            "catkin_pkg",
-            "rospkg",
-        ],
-        "waymo": [
-            "waymo-open-dataset-tf-2-2-0",
-        ],
+        "camera-obs": ["Panda3D==1.10.9", "panda3d-gltf==0.13"],
+        "ros": ["catkin_pkg", "rospkg"],
+        "waymo": ["waymo-open-dataset-tf-2-2-0"],
+        "extras": ["pynput>=1.7.4"],  # Used by HumanKeyboardAgent
```

Comment:
Seeing this now, I wonder if it would be best to remove the `human_in_the_loop` example and move the agent to the zoo. Then this requirement can sit in the agent's requirements.